bpf/node_config.h

```diff
@@ -115,7 +115,9 @@
 #define SNAT_COLLISION_RETRIES 32
 #endif
 
-#define EVENTS_MAP_RATE_LIMIT 0
+#ifndef EVENTS_MAP_RATE_LIMIT
```

Comment:
Why is the #ifndef now needed?

Reply:
Ok, thanks for the explanation! 

@ti-mo IIRC there's not a new strategy to migrate config values that are used in the complexity tests like this yet correct? So are changes like this to node_config.h still acceptable or is there another solution we can follow in such cases?

Reply:
@rgo3 Thanks for the ping! Correct, we don't have runtime configuration for verifier tests yet. These kinds of changes are fine, it's mostly altering existing behaviour. We only want to push back on new code using compile-time configuration here.